src/components/layout/Space.tsx

```diff
@@ -89,15 +89,18 @@ export const Space = React.memo((props: SpaceProps) => {
   const rightFill = boolify(alignLeft, alignHorizontal, alignCenter)
 
   // Margins:
-  const marginRemStyle = useMarginRemStyle({
-    bottomRem,
-    leftRem,
-    rightRem,
-    topRem,
-    aroundRem,
-    horizontalRem,
-    verticalRem
-  })
+  const marginRemStyle = useMarginRemStyle(
+    {
```

Comment:
Honestly I'd really prefer this entire file to go away. Similar to styled
components, all we get here is obfuscation.

Reply:
A cleanup for sure. I'll mark it as deprecated.